app/views/layouts/application.html.slim

```diff
@@ -27,6 +27,10 @@ html
               a#sidenavToggler.nav-link.text-center
                 i.fa.fa-fw.fa-angle-left
           ul.navbar-nav.ml-auto
+            span.balance
+              b
+                | Balance:
+              = current_user.transactions.map{|tr| tr.sum.to_i}.sum
```

Comment:
@marderer what do you think about renaming transaction's `sum` column to `amount`? It sounds more consistent in our case. I'm sorry that I didn't suggest it earlier.

Reply:
@marderer what do you think about moving sum calculation to `Transaction` model? Kind of logic is overkill for views. The main purpose of view - render, not calculate

```ruby
transaction.rb

def self.sum
  sum(&:amount)
end
```

Reply:
@marderer before adding this method try to write a test for it in `spec/unit/models/transaction_spec.rb`

Reply:
notice that all custom model's methods have to be tested

Reply:
@AleksSenkou  I agree with you, but I think that I should move this method to User model.